base/intset.jl

```diff
@@ -274,7 +274,7 @@ function ==(s1::IntSet, s2::IntSet)
             return false
         end
     end
-    filln = s1.fill1s ? UInt32(-1) : UInt32(0)
+    filln = s1.fill1s ? reinterpret(UInt32, Int32(-1)) : UInt32(0)
```

Comment:
Hah, even stronger evidence that nobody is using complements (at least not on master).

I would spell this `0xffffffff` or `-1 % UInt32`, but it's not a big deal since Julia/LLVM can figure all three out and inlines them.